### ISSUE TITLE
People properties are not converted to UTC for TimeWithZone values

### ISSUE THREAD

**Issue by wireframe** [mixpanel/mixpanel-ruby#81]
When setting a user property with a Rails DateTime attribute, mixpanel is save save the incorrect time in the user profile.

The [mixpanel-ruby fix_property_dates method](https://github.com/mixpanel/mixpanel-ruby/blob/a9c3f5c05eea3bfc3bf1508f06d8536d74d94be3/lib/mixpanel-ruby/people.rb#L242) is only converting dates to UTC if the object responds to `new_offset` and the `ActiveSupport::TimeWithZone` class does not.

sample code to reproduce the issue:
```ruby
MY_TOKEN = 'insert token here'
DISTINCT_ID = 'unique people id'
zone = 'Eastern Time (US & Canada)'
tracker = Mixpanel::Tracker.new(MY_TOKEN)

model = MyModel.find(some_id)
time = model.created_at.in_time_zone(zone)
#=> ActiveSupport::TimeWithZone
time.respond_to?(:new_offset)
#=> false

Time.use_zone(zone) do
  # time is being saved in mixpanel using local time instead of UTC
  tracker.people.set(DISTINCT_ID, test_property: time)
end
```

**Comment by eljojo** [comment]
Hi, I am experiencing the same problem.

In previous tests, manually converting my ``ActiveSupport::TimeWithZone``s sending ``#utc`` somehow did the job, but now it's broken.

Scenario:
- My rails app has the timezone set to Berlin
- mixpanel configuration as well
- it's 12:00 Berlin time

```ruby
tracker.people.set(user.id, {test_variable: Time.zone.now.utc})
```
sets following:
![screen shot 2015-06-11 at 12 03 55](https://cloud.githubusercontent.com/assets/491230/8104631/9849da1a-1032-11e5-92fa-1d05c8a63b72.png)

**Comment by eljojo** [comment]
Ping @joeatwork @zyskowsk

**Comment by zyskowsk** [comment]
Hey @wireframe!  Thanks for pointing this out, I will look into this and try to put up a fix asap

**Comment by zyskowsk** [comment]
fixed in v2.2.0 #88
